src/s1reader/s1_orbit.py

```diff
@@ -278,12 +332,8 @@ def get_orbit_file_from_dir(zip_path: str, orbit_dir: str, auto_download: bool =
             orbit_file = ''
```

Comment:
I know this is not from your PR. I personally don't like to use the variable `orbit_file` later in the code from a variable that is also used for the `for` loop, but I guess it's OK. In any case, this `else` is not necessary. The assignment `orbit_file = ''` can be done right after the break.

Reply:
On the same context as above. Please see the changes I've made.